Orange/widgets/tests/base.py

```diff
@@ -364,6 +364,26 @@ def modifiers(self, modifiers):
         finally:
             QTest.keyRelease(self.widget, Qt.Key_BassBoost, old_modifiers)
 
+    def test_minimum_size(self):
+        widget = getattr(self, "widget", None)
+        if widget is None:
+            return
```

Comment:
Or skip the tests using self.skipTest("minimum size needs to be tested manually")?

Otherwise, it will be marked as passing, even if it does not do anything.